### ISSUE TITLE
berry fetches region in wrong order from configs

### ISSUE THREAD

**Issue by okarahan** [zalando-stups/berry#16]
If I execute berry with a given berry.yaml and the region specified in it and in the same time I have a default region set in my ~/.aws/config file I expect that berry will take the region specified in the berry.yaml. Insteda of this berry uses the default region in my ~/.aws/config. This leads to errors if I try to get credentials from a mint bucket in a different region.

**Comment by hjacobs** [comment]
I think setting `region` in the berry.yaml has no effect anymore (this contradicts what http://docs.stups.io/en/latest/components/berry.html says). The bucket region should now be automatically determined (implemented by @tuxlife ).

**Comment by okarahan** [comment]
I only recognized the following:

If I execute berry and have a default region set in my ~/.aws/config and the default region doesn't match the region where the mint bucket is in then I get an error. If I remove my settings under ~/.aws/config then it works. But for the other tools like senza I prefer these settings under ~/.aws/config. Otherwise I had to provide the region for all commands.

**Comment by hjacobs** [comment]
@okarahan which version are you using? Berry should automatically use the right region depending on S3 bucket.

**Comment by okarahan** [comment]
How can I display the berry version ? It is not working via command line. As I know I recently updated all the stups tools.

**Comment by hjacobs** [comment]
@okarahan berry is not part of the `stups` meta package (as the `stups` package only includes end user tools). You can upgrade with `pip3 install -U stups-berry`.

**Comment by okarahan** [comment]
Ok sorry, with the new version it works as expected. I thought it belongs to the STUPS tools and expected that it has the recent version. Thank you @hjacobs

**Comment by hjacobs** [comment]
:+1:
